lasagne/init.py

```diff
@@ -132,6 +279,16 @@ def sample(self, shape):
 
 
 class Sparse(Initializer):
+    """Initialize weights as sparse matrix.
+
+    Parameters
+    ----------
+    sparsity : float
+        Fraction of weights with non-zero value. Larger value gives less
+        sparsity.
```

Comment:
> I think sparsity is an ok name, I find nonzeros or fraction less descriptive.

It's just that the meaning is inverted in that setting `sparsity` to a large value will give something not very sparse at all. That's a bit confusing.